src/reducers/palettes.test.js

```diff
@@ -8,10 +8,29 @@ describe('palettes', () => {
     expect(result).toEqual(expected)
   })
 
+  it('should set state with the passed palettes if the type is SET_PALETTES', () => {
+    const initialState = []
+    const mockPalettes = [ { palette_name: 'palette one'}, { palette_name: 'palette two'} ]
```

Comment:
We can consider refactoring these variables into a beforeEach to simplify the it blocks.